tajo-catalog/tajo-catalog-server/src/test/java/org/apache/tajo/catalog/TestCatalog.java

```diff
@@ -274,10 +274,8 @@ public void testDropDatabaseWithAllTables() throws Exception {
     Map<String, List<String>> createdTablesMap = createBaseDatabaseAndTables();
 
     // Each time we drop one database, check all databases and their tables.
-    Iterator<String> it = new ArrayList<>(createdTablesMap.keySet()).iterator();
-    while(it.hasNext()) {
+    for (String databaseName : new ArrayList<>(createdTablesMap.keySet())) {
```

Comment:
Thanks for your reply. 
As my understanding, That line is written to convert Set to List because Set has no ordering. I have no idea with another way to get same result. Can I get some more details about your comment?